tests/test_api.py

```diff
@@ -30,7 +30,7 @@
 
 def test_match_raises_if_the_configuration_does_not_have_all_the_keys():
     config = {
-        'doc_type': 'records',
+        'doc_type': 'hep',
```

Comment:
I copied these tests to run some queries but didn't get any results, it took some time figuring out that the `doc_type` was wrong.

Reply:
Whoops! :dancer: